pac/atsame70q21b/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "atsame70q21b"
```

Comment:
I'm not keen to add each particular chip variant separately, rather for the whole family.